pipeline/patch_VAE.py

```diff
@@ -372,30 +371,27 @@ def process_VAE(raw_folder: str,
     # For inference same normalization parameters can be used or determined from the inference data,
     # depending on if the inference data has the same distribution as training data
 
-    model_path = config_.inference.weights
-    # weights_dir = config_.files.weights_dir
-    channels = config_.inference.channels
-    num_hiddens = config_.training.num_hiddens
-    num_residual_hiddens = config_.training.num_residual_hiddens
-    num_embeddings = config_.training.num_embeddings
-    commitment_cost = config_.training.commitment_cost
-    network = config_.inference.model
-    save_output = config_.inference.save_output
+    model_path = config_.latent_encoding.weights
+    channels = config_.latent_encoding.channels
+
+    num_hiddens = config_.latent_encoding.num_hiddens
+    num_residual_hiddens = config_.latent_encoding.num_residual_hiddens
+    num_embeddings = config_.latent_encoding.num_embeddings
+    commitment_cost = config_.latent_encoding.commitment_cost
+
+    network = config_.latent_encoding.model
+    save_output = config_.latent_encoding.save_output
 
     assert len(channels) > 0, "At least one channel must be specified"
 
     # these sites should be from a single condition (C5, C4, B-wells, etc..)
-    model_dir = os.path.dirname(model_path)
-    #TODO: add model_name to the config. Set the default to be the same as model folder name
-    model_name = os.path.basename(model_dir)
-    # output_dir = os.path.join(summary_folder, model_name + '_pool_norm')
-    output_dir = os.path.join(raw_folder, model_name)
-    os.makedirs(output_dir, exist_ok=True)
 
-    assert len(set(site[:2] for site in sites)) == 1, \
-        "Sites should be from a single well/condition"
-    well = sites[0][:2]
-    # TODO: expose normalization parameters in train config
+    # model_path = os.path.join(weights_dir, 'model.pt')
+    # model_name = os.path.basename(weights_dir)
+    # output_dir = os.path.join(raw_folder, model_name)
+    # os.makedirs(output_dir, exist_ok=True)
+    output_dir = raw_folder
```

Comment:
yes that's right.  How do we want to approach this?  It will cause some problems for `run_dim_reduction` which looks at the raw folder for this.  Now that we define a raw/supp/weights path for every stage of the pipeline, could we generalize and say "input/output"?  Then, for example, the output of `run_vae.process` will be the input of `run_dim_reduction`?

Reply:
the encoded vectors will depends on the `raw` and `weight`. So the way I deal with this is creating model folder that has the same name as weight folder inside each raw folder (the commentated out code), but there might be easier solution

Reply:
Ok i think i like your solution.  Will switch over to this.

Another option is to simply write the file name to include the model name (and maybe some other unique identifier).  But this causes other issues with `run_dim_reduction` which hardcodes the filename format it searches for.